src/cloudflare/internal/test/ai/ai-api-test.wd-test

```diff
@@ -0,0 +1,36 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "ai-api-test",
+      worker = (
+        modules = [
+          (name = "worker", esModule = embed "ai-api-test.js")
+        ],
+        compatibilityDate = "2023-01-15",
+        compatibilityFlags = ["nodejs_compat"],
+        bindings = [
+        (
+          name = "ai",
+          wrapped = (
+            moduleName = "cloudflare-internal:ai-api",
```

Comment:
hmm, `internal` implies it shouldn't be used by end users. But this isn't the case here, right? Maybe this shouldn't be an internal module?

Reply:
agreed, i've just moved this module to outside the internal folder

Reply:
humm, but end users shouldn't be able to initialize the Ai class on themself, it should be done by wrangler automatically when they define the AI binding
I've kept the code in the internal folder, but added a mapping outside to end users could import the options and Ai class from `cloudflare:ai`

Reply:
What do other APIs do? What is the pattern we want people to use for this @mikea ?

Reply:
i'm currently having some issues importing this module from wrangler iv'e tried with `cloudflare:ai` and `cloudflare-internal:ai-api`
![image](https://github.com/cloudflare/workerd/assets/5445926/22dd40fb-dc66-4236-b129-f9fb59dc1d48)